sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -372,6 +374,16 @@ private[hive] class SparkExecuteStatementOperation(
       }
     }
   }
+
+  private def prepareLogLevel(): Unit = {
+    val debugClasses = sqlContext.conf.debugClasses()
```

Comment:
this configuration seems only affecting thriftserver. it should have an effect in regular Spark session as well. But more importantly, why don't you just restart or start another one for debugging purpose? I don't think this configuration is worth much.

Reply:
We just want to change the log level of thriftserver when our application has already deployed. If it doesn't support dynamic log level, we should change the log4j file and restart the thrift server every time. It is not convenient.